packages/services/core-analytics/.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+dist
+harness
```

Comment:
Can't find any ref to harness. Is it used?

Reply:
Good spot, I took that folder out, I will remove